\documentclass[a4paper,11pt,oneside]{article}
\usepackage{graphicx}
\usepackage{url}
\usepackage{multirow}
\usepackage{rotating}
\usepackage{color}
\usepackage[compact]{titlesec}
\usepackage[portrait,top=1cm, bottom=2cm, left=1cm, right=1cm]{geometry}
\usepackage{float}
\restylefloat{table}
\begin{document}
\renewcommand*{\familydefault}{\sfdefault}
\normalfont
\section*{\large{NanoOK report for main}}
\subsection*{Pass and fail counts}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c}
{\bf Type} & {\bf Pass} & {\bf Fail} \\
Template & 500 & 0  \\
Complement & 500 & 0  \\
2D & 500 & 0  \\
\end{tabular}
}
\end{table}
\subsection*{Read lengths}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c c c c c c c c}
{\bf Type} & {\bf NumReads} & {\bf TotalBases} & {\bf Mean} & {\bf Longest} & {\bf Shortest} & {\bf N50} & {\bf N50Count} & {\bf N90} & {\bf N90Count} \\
Template & 500 & 1685124 & 3370.25 & 10794 & 260 & 4309 & 144 & 1969 & 354 \\
Complement & 500 & 1788780 & 3577.56 & 12018 & 300 & 4533 & 146 & 2014 & 358 \\
2D & 500 & 1824129 & 3648.26 & 12165 & 297 & 4672 & 145 & 2117 & 355 \\
\end{tabular}
}
\end{table}
\vspace{-10mm}
\begin{figure}[H]
\centering
\includegraphics[width=.3\linewidth]{/home/data/inputs/nanook/main/graphs_last_passonly/all_Template_lengths.pdf}
\includegraphics[width=.3\linewidth]{/home/data/inputs/nanook/main/graphs_last_passonly/all_Complement_lengths.pdf}
\includegraphics[width=.3\linewidth]{/home/data/inputs/nanook/main/graphs_last_passonly/all_2D_lengths.pdf}
\end{figure}
\subsection*{Template alignments}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c}
Number of reads & 500 & \\
Number of reads with alignments & 471 & (94.20\%) \\
Number of reads without alignments & 29 & (5.80\%) \\
\end{tabular}
}
\end{table}
\vspace{-10mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c c c c c c}
          &             & {\bf Number of} & {\bf \% of} & {\bf Mean read} & {\bf Aligned} & {\bf Mean} & {\bf Longest} \\
{\bf ID} & {\bf Size} & {\bf Reads}     & {\bf Reads}  & {\bf length}    & {\bf bases}   & {\bf coverage} & {\bf Perf Kmer} \\
DNA CS & 3560 & 17 & 3.40 & 3018.24 & 57853 & 16.25 & 47 \\
Escherichia coli & 4686137 & 454 & 90.80 & 3492.26 & 1766480 & 0.38 & 59 \\
\end{tabular}
}
\end{table}
\subsection*{Complement alignments}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c}
Number of reads & 500 & \\
Number of reads with alignments & 479 & (95.80\%) \\
Number of reads without alignments & 21 & (4.20\%) \\
\end{tabular}
}
\end{table}
\vspace{-10mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c c c c c c}
          &             & {\bf Number of} & {\bf \% of} & {\bf Mean read} & {\bf Aligned} & {\bf Mean} & {\bf Longest} \\
{\bf ID} & {\bf Size} & {\bf Reads}     & {\bf Reads}  & {\bf length}    & {\bf bases}   & {\bf coverage} & {\bf Perf Kmer} \\
DNA CS & 3560 & 17 & 3.40 & 3252.59 & 57967 & 16.28 & 47 \\
Escherichia coli & 4686137 & 462 & 92.40 & 3660.72 & 1801633 & 0.38 & 47 \\
\end{tabular}
}
\end{table}
\subsection*{2D alignments}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c}
Number of reads & 500 & \\
Number of reads with alignments & 486 & (97.20\%) \\
Number of reads without alignments & 14 & (2.80\%) \\
\end{tabular}
}
\end{table}
\vspace{-10mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c c c c c c}
          &             & {\bf Number of} & {\bf \% of} & {\bf Mean read} & {\bf Aligned} & {\bf Mean} & {\bf Longest} \\
{\bf ID} & {\bf Size} & {\bf Reads}     & {\bf Reads}  & {\bf length}    & {\bf bases}   & {\bf coverage} & {\bf Perf Kmer} \\
DNA CS & 3560 & 17 & 3.40 & 3312.65 & 59003 & 16.57 & 87 \\
Escherichia coli & 4686137 & 469 & 93.80 & 3692.05 & 1814283 & 0.39 & 182 \\
\end{tabular}
}
\end{table}
\clearpage
\subsection*{DNA CS error analysis}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c c}
 & Template & Complement & 2D \\
Overall base identity (excluding indels) & 76.00\% & 70.31\% & 86.06\% \\
Aligned base identity (excluding indels) & 80.90\% & 80.23\% & 91.54\% \\
Identical bases per 100 aligned bases (including indels) & 67.41\% & 67.07\% & 82.14\% \\
Inserted bases per 100 aligned bases (including indels) & 3.00\% & 5.09\% & 3.89\% \\
Deleted bases per 100 aligned bases (including indels) & 13.67\% & 11.31\% & 6.37\% \\
Substitutions per 100 aligned bases (including indels) & 15.92\% & 16.53\% & 7.59\% \\
Mean insertion size & 1.50 & 1.61 & 1.49 \\
Mean deletion size & 1.66 & 1.65 & 1.55 \\
\end{tabular}
}
\end{table}
\vspace{-5mm}
\begin{figure}[H]
\centering
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_insertions.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_insertions.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_insertions.pdf} \\
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_deletions.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_deletions.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_deletions.pdf}
\end{figure}
\subsection*{DNA CS read identity}
\vspace{-3mm}
\begin{figure}[H]
\centering
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_length_vs_identity_hist.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_length_vs_identity_hist.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_length_vs_identity_hist.pdf} \\
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_length_vs_identity_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_length_vs_identity_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_length_vs_identity_scatter.pdf} \\
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_read_fraction_vs_alignment_identity_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_read_fraction_vs_alignment_identity_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_read_fraction_vs_alignment_identity_scatter.pdf}
\end{figure}
\begin{figure}[H]
\centering
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_percent_aligned_vs_length_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_percent_aligned_vs_length_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_percent_aligned_vs_length_scatter.pdf}
\end{figure}
\subsection*{DNA CS perfect kmers}
\vspace{-3mm}
\begin{figure}[H]
\centering
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_cumulative_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_cumulative_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_cumulative_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_best_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_best_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_best_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_longest_perfect_vs_length_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_longest_perfect_vs_length_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_longest_perfect_vs_length_scatter.pdf}
\end{figure}
\subsection*{DNA CS coverage}
\vspace{-3mm}
\begin{figure}[H]
\centering
\includegraphics[height=2cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_coverage.pdf} \\
\includegraphics[height=2cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_coverage.pdf} \\
\includegraphics[height=2cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_coverage.pdf} \\
\includegraphics[height=2cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_gc.pdf}
\end{figure}
\subsection*{DNA CS 5-mer analysis}
\subsection*{Under-represented 5-mers}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{7pt}{9pt}\selectfont
\begin{tabular}{|c|c c c c|c c c c|c c c c|}
\cline{1-13}
 & \multicolumn{4}{c|}{Template} & \multicolumn{4}{c|}{Complement} & \multicolumn{4}{c|}{2D} \\
Rank & kmer & Ref \% & Read \% & Diff \% & kmer & Ref \% & Read \% & Diff \% & kmer & Ref \% & Read \% & Diff \% \\
\cline{1-13}
1 & TTTTT & 0.759 & 0.078 & -0.681 & TTTTT & 0.759 & 0.114 & -0.645 & TTTTT & 0.759 & 0.039 & -0.720 \\
2 & AAAAA & 0.478 & 0.057 & -0.421 & AAAAA & 0.478 & 0.080 & -0.398 & AAAAA & 0.478 & 0.055 & -0.423 \\
3 & TGATG & 0.393 & 0.137 & -0.257 & AAAAC & 0.337 & 0.125 & -0.212 & TGATG & 0.393 & 0.203 & -0.191 \\
4 & AAAAC & 0.337 & 0.082 & -0.255 & GATGT & 0.309 & 0.100 & -0.209 & CTTTT & 0.253 & 0.091 & -0.162 \\
5 & GATGT & 0.309 & 0.096 & -0.213 & TGATG & 0.393 & 0.199 & -0.194 & GATGT & 0.309 & 0.148 & -0.161 \\
6 & AATAT & 0.309 & 0.111 & -0.198 & GTTTT & 0.281 & 0.114 & -0.167 & AAAAC & 0.337 & 0.180 & -0.158 \\
7 & CTGAT & 0.309 & 0.123 & -0.186 & GCAAT & 0.309 & 0.152 & -0.157 & CTGAT & 0.309 & 0.155 & -0.154 \\
8 & GTTTT & 0.281 & 0.105 & -0.176 & AACAA & 0.281 & 0.125 & -0.156 & GCAAT & 0.309 & 0.158 & -0.151 \\
9 & CAAAA & 0.225 & 0.059 & -0.166 & AGAAA & 0.281 & 0.125 & -0.156 & TTATC & 0.309 & 0.165 & -0.144 \\
10 & TAAAA & 0.225 & 0.064 & -0.160 & ATTAA & 0.281 & 0.132 & -0.149 & TGTGA & 0.225 & 0.084 & -0.141 \\
\cline{1-13}
\end{tabular}
}
\end{table}
\vspace{-3mm}
\subsection*{Over-represented 5-mers}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{7pt}{9pt}\selectfont
\begin{tabular}{|c|c c c c|c c c c|c c c c|}
\cline{1-13}
 & \multicolumn{4}{c|}{Template} & \multicolumn{4}{c|}{Complement} & \multicolumn{4}{c|}{2D} \\
Rank & kmer & Ref \% & Read \% & Diff \% & kmer & Ref \% & Read \% & Diff \% & kmer & Ref \% & Read \% & Diff \% \\
\cline{1-13}
1 & TCAGC & 0.028 & 0.184 & 0.155 & GAGAG & 0.112 & 0.306 & 0.194 & ATCAG & 0.056 & 0.212 & 0.155 \\
2 & ATCAG & 0.056 & 0.197 & 0.141 & GAGGA & 0.000 & 0.170 & 0.170 & CATCA & 0.112 & 0.251 & 0.138 \\
3 & TCGGG & 0.028 & 0.166 & 0.138 & GTGTG & 0.000 & 0.167 & 0.167 & TCAGC & 0.028 & 0.162 & 0.134 \\
4 & GAGGA & 0.000 & 0.125 & 0.125 & AGAGA & 0.140 & 0.299 & 0.158 & GCATC & 0.084 & 0.210 & 0.126 \\
5 & CATCT & 0.000 & 0.125 & 0.125 & GGAGA & 0.028 & 0.178 & 0.149 & AACCA & 0.028 & 0.148 & 0.120 \\
6 & AACGT & 0.084 & 0.203 & 0.119 & CTTTG & 0.028 & 0.161 & 0.133 & CAGCA & 0.056 & 0.171 & 0.115 \\
7 & CGTAG & 0.028 & 0.146 & 0.118 & AGGGG & 0.000 & 0.132 & 0.132 & ATCCA & 0.028 & 0.123 & 0.095 \\
8 & CTTTG & 0.028 & 0.144 & 0.116 & GAGAA & 0.084 & 0.214 & 0.129 & CATCT & 0.000 & 0.092 & 0.092 \\
9 & GGGGA & 0.028 & 0.143 & 0.114 & TACTT & 0.000 & 0.127 & 0.127 & TTCCT & 0.084 & 0.174 & 0.090 \\
10 & TCCGT & 0.056 & 0.170 & 0.114 & TCGGG & 0.028 & 0.152 & 0.124 & ACCAA & 0.028 & 0.117 & 0.089 \\
\cline{1-13}
\end{tabular}
}
\end{table}
\vspace{-8mm}
\begin{figure}[H]
\centering
\includegraphics[height=8cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_kmer_scatter.pdf} 
\includegraphics[height=8cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_kmer_scatter.pdf} \\
\includegraphics[height=8cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_kmer_scatter.pdf} \\
\end{figure}
\subsection*{DNA CS GC content}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Template_GC_hist.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_Complement_GC_hist.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/DNA_CS/DNA_CS_2D_GC_hist.pdf}
\clearpage
\subsection*{Escherichia coli error analysis}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{9pt}{11pt}\selectfont
\begin{tabular}{l c c c}
 & Template & Complement & 2D \\
Overall base identity (excluding indels) & 74.62\% & 72.29\% & 86.31\% \\
Aligned base identity (excluding indels) & 80.51\% & 81.26\% & 91.72\% \\
Identical bases per 100 aligned bases (including indels) & 66.97\% & 67.86\% & 82.38\% \\
Inserted bases per 100 aligned bases (including indels) & 3.89\% & 5.49\% & 4.58\% \\
Deleted bases per 100 aligned bases (including indels) & 12.92\% & 10.99\% & 5.60\% \\
Substitutions per 100 aligned bases (including indels) & 16.22\% & 15.65\% & 7.44\% \\
Mean insertion size & 1.59 & 1.65 & 1.60 \\
Mean deletion size & 1.68 & 1.64 & 1.50 \\
\end{tabular}
}
\end{table}
\vspace{-5mm}
\begin{figure}[H]
\centering
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_insertions.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_insertions.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_insertions.pdf} \\
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_deletions.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_deletions.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_deletions.pdf}
\end{figure}
\subsection*{Escherichia coli read identity}
\vspace{-3mm}
\begin{figure}[H]
\centering
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_length_vs_identity_hist.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_length_vs_identity_hist.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_length_vs_identity_hist.pdf} \\
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_length_vs_identity_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_length_vs_identity_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_length_vs_identity_scatter.pdf} \\
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_read_fraction_vs_alignment_identity_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_read_fraction_vs_alignment_identity_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_read_fraction_vs_alignment_identity_scatter.pdf}
\end{figure}
\begin{figure}[H]
\centering
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_percent_aligned_vs_length_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_percent_aligned_vs_length_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_percent_aligned_vs_length_scatter.pdf}
\end{figure}
\subsection*{Escherichia coli perfect kmers}
\vspace{-3mm}
\begin{figure}[H]
\centering
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_cumulative_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_cumulative_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_cumulative_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_best_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_best_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_best_perfect_kmers.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_longest_perfect_vs_length_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_longest_perfect_vs_length_scatter.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_longest_perfect_vs_length_scatter.pdf}
\end{figure}
\subsection*{Escherichia coli coverage}
\vspace{-3mm}
\begin{figure}[H]
\centering
\includegraphics[height=2cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_coverage.pdf} \\
\includegraphics[height=2cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_coverage.pdf} \\
\includegraphics[height=2cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_coverage.pdf} \\
\includegraphics[height=2cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_gc.pdf}
\end{figure}
\subsection*{Escherichia coli 5-mer analysis}
\subsection*{Under-represented 5-mers}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{7pt}{9pt}\selectfont
\begin{tabular}{|c|c c c c|c c c c|c c c c|}
\cline{1-13}
 & \multicolumn{4}{c|}{Template} & \multicolumn{4}{c|}{Complement} & \multicolumn{4}{c|}{2D} \\
Rank & kmer & Ref \% & Read \% & Diff \% & kmer & Ref \% & Read \% & Diff \% & kmer & Ref \% & Read \% & Diff \% \\
\cline{1-13}
1 & AAAAA & 0.246 & 0.048 & -0.198 & CGCCA & 0.285 & 0.079 & -0.206 & TTTTT & 0.251 & 0.036 & -0.215 \\
2 & TTTTT & 0.251 & 0.063 & -0.188 & TTTTT & 0.251 & 0.069 & -0.182 & AAAAA & 0.246 & 0.036 & -0.210 \\
3 & CGCCA & 0.285 & 0.106 & -0.179 & AAAAA & 0.246 & 0.067 & -0.180 & TGGCG & 0.275 & 0.179 & -0.096 \\
4 & CGCTG & 0.259 & 0.089 & -0.170 & GCCAG & 0.279 & 0.124 & -0.156 & CGCCA & 0.285 & 0.192 & -0.093 \\
5 & GCCAG & 0.279 & 0.133 & -0.147 & CCAGC & 0.287 & 0.134 & -0.153 & AAAAT & 0.194 & 0.105 & -0.089 \\
6 & CGCCG & 0.219 & 0.091 & -0.128 & CTGGC & 0.279 & 0.140 & -0.139 & CAAAA & 0.169 & 0.087 & -0.082 \\
7 & CTGGC & 0.279 & 0.155 & -0.124 & CAGCA & 0.262 & 0.136 & -0.126 & GCCAG & 0.279 & 0.200 & -0.079 \\
8 & GCCGC & 0.209 & 0.087 & -0.122 & TCGCC & 0.203 & 0.081 & -0.122 & CTGGC & 0.279 & 0.201 & -0.078 \\
9 & ACGCC & 0.176 & 0.054 & -0.121 & TGGCG & 0.275 & 0.154 & -0.121 & GCTGG & 0.279 & 0.206 & -0.073 \\
10 & GCTGG & 0.279 & 0.159 & -0.120 & CGCTG & 0.259 & 0.139 & -0.120 & TTTTG & 0.172 & 0.099 & -0.073 \\
\cline{1-13}
\end{tabular}
}
\end{table}
\vspace{-3mm}
\subsection*{Over-represented 5-mers}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{7pt}{9pt}\selectfont
\begin{tabular}{|c|c c c c|c c c c|c c c c|}
\cline{1-13}
 & \multicolumn{4}{c|}{Template} & \multicolumn{4}{c|}{Complement} & \multicolumn{4}{c|}{2D} \\
Rank & kmer & Ref \% & Read \% & Diff \% & kmer & Ref \% & Read \% & Diff \% & kmer & Ref \% & Read \% & Diff \% \\
\cline{1-13}
1 & CGGGG & 0.055 & 0.166 & 0.111 & GAGAG & 0.045 & 0.255 & 0.209 & CAAAT & 0.105 & 0.153 & 0.048 \\
2 & TCGGG & 0.059 & 0.153 & 0.094 & AGAGA & 0.071 & 0.237 & 0.166 & AAGGA & 0.056 & 0.099 & 0.043 \\
3 & GTCGT & 0.078 & 0.171 & 0.093 & CGGGG & 0.055 & 0.156 & 0.101 & CTCGT & 0.043 & 0.081 & 0.038 \\
4 & GGGGC & 0.060 & 0.149 & 0.089 & GGGGC & 0.060 & 0.159 & 0.099 & AGGCA & 0.093 & 0.131 & 0.038 \\
5 & CGTAG & 0.058 & 0.142 & 0.084 & GGGGG & 0.031 & 0.129 & 0.097 & AGCAT & 0.102 & 0.138 & 0.036 \\
6 & TCGTA & 0.053 & 0.136 & 0.083 & CTGAG & 0.050 & 0.133 & 0.083 & CTAGC & 0.008 & 0.044 & 0.036 \\
7 & GGGAG & 0.042 & 0.118 & 0.076 & GCTAG & 0.008 & 0.089 & 0.081 & TAAAT & 0.112 & 0.147 & 0.035 \\
8 & CGTCG & 0.114 & 0.189 & 0.075 & GAGGG & 0.031 & 0.111 & 0.080 & GCATG & 0.090 & 0.125 & 0.035 \\
9 & GGGGG & 0.031 & 0.106 & 0.075 & TCGGG & 0.059 & 0.136 & 0.076 & TCTAG & 0.004 & 0.038 & 0.034 \\
10 & ACGGG & 0.069 & 0.143 & 0.074 & AGGGG & 0.039 & 0.114 & 0.074 & TCCGT & 0.066 & 0.100 & 0.034 \\
\cline{1-13}
\end{tabular}
}
\end{table}
\vspace{-8mm}
\begin{figure}[H]
\centering
\includegraphics[height=8cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_kmer_scatter.pdf} 
\includegraphics[height=8cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_kmer_scatter.pdf} \\
\includegraphics[height=8cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_kmer_scatter.pdf} \\
\end{figure}
\subsection*{Escherichia coli GC content}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Template_GC_hist.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_Complement_GC_hist.pdf}
\includegraphics[height=3.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/Escherichia_coli/Escherichia_coli_2D_GC_hist.pdf}
\clearpage
\subsection*{All reference 21mer analysis}
\vspace{-3mm}
\begin{figure}[H]
\centering
\includegraphics[width=.3\linewidth]{/home/data/inputs/nanook/main/graphs_last_passonly/all_Template_21mers.pdf}
\includegraphics[width=.3\linewidth]{/home/data/inputs/nanook/main/graphs_last_passonly/all_Complement_21mers.pdf}
\includegraphics[width=.3\linewidth]{/home/data/inputs/nanook/main/graphs_last_passonly/all_2D_21mers.pdf}
\end{figure}
\subsection*{All reference substitutions}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{8pt}{10pt}\selectfont
\begin{tabular}{|c c|c c c c|c c c c|c c c c|}
\hline
 & & \multicolumn{4}{c|}{Template substituted \%} & \multicolumn{4}{c|}{Complement substituted \%} & \multicolumn{4}{c|}{2D substituted \%} \\
 & & a & c & g & t & a & c & g & t & a & c & g & t \\
\hline
\multirow{4}{*}{\rotatebox[origin=c]{90}{Reference}} & A & 0.00 & 8.60 & 9.72 & 5.00 & 0.00 & 8.92 & 9.08 & 5.18 & 0.00 & 8.39 & 8.45 & 4.45\\
 & C & 8.44 & 0.00 & 8.83 & 9.38 & 9.42 & 0.00 & 8.62 & 8.86 & 9.31 & 0.00 & 10.04 & 9.25\\
 & G & 9.41 & 8.91 & 0.00 & 8.50 & 8.86 & 8.57 & 0.00 & 9.31 & 9.32 & 9.94 & 0.00 & 9.51\\
 & T & 4.95 & 9.78 & 8.48 & 0.00 & 5.28 & 8.98 & 8.93 & 0.00 & 4.55 & 8.30 & 8.52 & 0.00\\
\hline
\end{tabular}
}
\end{table}
\subsection*{Kmer motifs before errors}
\subsection*{3-mer error motif analysis}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{6pt}{8pt}\selectfont
\tabcolsep=0.15cm
\begin{tabular}{|c|c c c|c c c|c c c|c}
\cline{1-10}
 & \multicolumn{3}{c|}{Template} & \multicolumn{3}{c|}{Complement} & \multicolumn{3}{c|}{2D} & \\
Rank & Insertion & Deletion & Substitution & Insertion & Deletion & Substitution & Insertion & Deletion & Substitution & \\
\cline{1-10}
1 & TTC (2.99\%) & AAA (3.42\%) & AAA (3.60\%) & AAA (2.93\%) & AAA (3.44\%) & AAA (3.75\%) & GCA (3.07\%) & AAA (5.68\%) & AAA (3.90\%) & \multirow{10}{*}{\rotatebox[origin=c]{90}{Most common}}\\
2 & GCA (2.59\%) & TTC (3.09\%) & TTC (3.54\%) & GCA (2.77\%) & GAA (2.96\%) & GCA (3.60\%) & AAA (2.80\%) & TTT (4.85\%) & GCA (3.62\%)\\
3 & TCA (2.57\%) & TGC (2.94\%) & GCA (3.27\%) & TGC (2.58\%) & TTT (2.74\%) & GAA (3.19\%) & TTC (2.71\%) & GCC (2.76\%) & GAA (3.26\%)\\
4 & AAA (2.57\%) & GCC (2.70\%) & GAA (2.82\%) & TTC (2.47\%) & GCA (2.64\%) & TTC (2.85\%) & GAA (2.63\%) & GAA (2.62\%) & TTC (2.80\%)\\
5 & ATC (2.51\%) & TTT (2.65\%) & TCA (2.56\%) & GAA (2.40\%) & TGC (2.58\%) & TTT (2.66\%) & TGC (2.55\%) & GCA (2.47\%) & TCA (2.62\%)\\
6 & TGC (2.44\%) & GCA (2.59\%) & TGC (2.53\%) & CAG (2.33\%) & TTC (2.39\%) & TGC (2.43\%) & TCA (2.52\%) & GCG (2.38\%) & GCC (2.61\%)\\
7 & TTT (2.29\%) & TCA (2.41\%) & ATC (2.30\%) & TCA (2.30\%) & GGC (2.38\%) & TCA (2.39\%) & CAG (2.43\%) & TGC (2.23\%) & TTT (2.54\%)\\
8 & GCC (2.23\%) & GAA (2.37\%) & GCC (2.28\%) & GGC (2.17\%) & GCC (2.35\%) & ATC (2.23\%) & ATC (2.34\%) & TCA (2.12\%) & GCG (2.42\%)\\
9 & CAG (2.18\%) & GCG (2.17\%) & TTT (2.20\%) & TTT (2.09\%) & CAG (2.27\%) & CCA (2.16\%) & GCC (2.25\%) & TTC (2.10\%) & GTT (2.32\%)\\
10 & GCG (2.17\%) & ACG (2.11\%) & ACG (2.19\%) & GCC (2.09\%) & AGC (2.10\%) & GGC (2.16\%) & CGC (2.24\%) & CAG (2.03\%) & TGC (2.24\%)\\
\cline{1-10}
\rule{0pt}{0.6cm}
  & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Template_top_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Template_top_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Template_top_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Complement_top_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Complement_top_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Complement_top_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_2D_top_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_2D_top_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_2D_top_k3.png} \\
\cline{1-10}
-10 & AGT (0.99\%) & GTA (0.95\%) & GGG (0.88\%) & GAC (1.02\%) & AGG (0.98\%) & CCC (0.97\%) & AGG (0.98\%) & CTC (0.93\%) & GTA (0.89\%) & \multirow{10}{*}{\rotatebox[origin=c]{90}{Least common}}\\
-9 & CCC (0.95\%) & GGG (0.95\%) & AGG (0.84\%) & GTG (1.02\%) & GGG (0.98\%) & CTT (0.91\%) & GGG (0.97\%) & GAG (0.93\%) & CTT (0.88\%)\\
-8 & GGG (0.95\%) & ACT (0.94\%) & TGT (0.84\%) & CTC (0.99\%) & AGT (0.95\%) & GGG (0.84\%) & TAT (0.97\%) & AGA (0.83\%) & CCT (0.88\%)\\
-7 & AGA (0.91\%) & CGA (0.91\%) & ACT (0.84\%) & ACT (0.99\%) & GTA (0.89\%) & AGG (0.82\%) & AGT (0.93\%) & GTA (0.82\%) & ACT (0.82\%)\\
-6 & CTT (0.84\%) & AGT (0.80\%) & AGT (0.83\%) & GGG (0.97\%) & CTC (0.86\%) & AGT (0.79\%) & TGT (0.92\%) & CGA (0.80\%) & CGA (0.79\%)\\
-5 & CTC (0.83\%) & GGA (0.76\%) & CTT (0.83\%) & GGA (0.93\%) & CCT (0.85\%) & GAG (0.72\%) & GAG (0.85\%) & CCT (0.69\%) & GAG (0.78\%)\\
-4 & AGG (0.81\%) & GAG (0.72\%) & AGA (0.80\%) & GAG (0.89\%) & GAG (0.81\%) & CCT (0.69\%) & AGA (0.83\%) & ACT (0.66\%) & AGA (0.64\%)\\
-3 & GAG (0.80\%) & AGA (0.71\%) & GAG (0.68\%) & AGG (0.77\%) & ACT (0.77\%) & ACT (0.65\%) & GGA (0.76\%) & GGA (0.58\%) & GGA (0.62\%)\\
-2 & TAG (0.47\%) & TAG (0.52\%) & CTA (0.40\%) & CTA (0.50\%) & CTA (0.53\%) & CTA (0.43\%) & TAG (0.49\%) & CTA (0.57\%) & TAG (0.46\%)\\
-1 & CTA (0.44\%) & CTA (0.42\%) & TAG (0.29\%) & TAG (0.44\%) & TAG (0.42\%) & TAG (0.36\%) & CTA (0.46\%) & TAG (0.55\%) & CTA (0.40\%)\\
\cline{1-10}
\rule{0pt}{0.6cm}
  & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Template_bottom_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Template_bottom_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Template_bottom_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Complement_bottom_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Complement_bottom_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Complement_bottom_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_2D_bottom_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_2D_bottom_k3.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_2D_bottom_k3.png} \\
\cline{1-10}
\end{tabular}
}
\end{table}
\vspace{-9mm}
{\fontsize{8}{8}\textsf{Kmer space for 3-mers: 64 \hspace{5mm} Random chance for any given 3-mer: 1.56\%}}
\vspace{5mm}
\subsection*{4-mer error motif analysis}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{6pt}{8pt}\selectfont
\tabcolsep=0.15cm
\begin{tabular}{|c|c c c|c c c|c c c|c}
\cline{1-10}
 & \multicolumn{3}{c|}{Template} & \multicolumn{3}{c|}{Complement} & \multicolumn{3}{c|}{2D} & \\
Rank & Insertion & Deletion & Substitution & Insertion & Deletion & Substitution & Insertion & Deletion & Substitution & \\
\cline{1-10}
1 & ATCA (1.10\%) & TGCC (1.09\%) & GAAA (1.18\%) & ATCA (1.02\%) & CAGC (1.02\%) & AGCA (1.04\%) & GGCA (0.97\%) & AAAA (1.81\%) & AAAA (1.08\%) & \multirow{10}{*}{\rotatebox[origin=c]{90}{Most common}}\\
2 & AACG (0.89\%) & AAAA (1.02\%) & TTTC (1.05\%) & CAGC (0.94\%) & CAAA (0.97\%) & GAAA (1.02\%) & GAAA (0.89\%) & TTTT (1.72\%) & GGCA (1.08\%)\\
3 & TGCC (0.89\%) & GAAA (1.00\%) & AAAA (1.03\%) & CTGC (0.86\%) & AAAA (0.91\%) & CAAA (0.96\%) & ATCA (0.89\%) & CAAA (1.70\%) & GAAA (1.05\%)\\
4 & GCTG (0.87\%) & TTTC (0.96\%) & ATCA (0.92\%) & CAAA (0.86\%) & TAAA (0.87\%) & AAAA (0.95\%) & CTGC (0.80\%) & TAAA (1.35\%) & ATCA (0.98\%)\\
5 & CATC (0.83\%) & CAAA (0.95\%) & CTTC (0.87\%) & AGCA (0.85\%) & GAAA (0.87\%) & CGCA (0.93\%) & CCAG (0.80\%) & ATTT (1.29\%) & TGCC (0.95\%)\\
6 & GAAA (0.82\%) & TTCA (0.90\%) & CGCA (0.86\%) & AAAA (0.77\%) & TGAA (0.86\%) & ATCA (0.91\%) & TGCC (0.80\%) & CTTT (1.13\%) & CAAA (0.93\%)\\
7 & TTTC (0.82\%) & ATCA (0.89\%) & GGCA (0.85\%) & GCGC (0.76\%) & ATTT (0.84\%) & GGCA (0.90\%) & CAAA (0.79\%) & GAAA (1.09\%) & TGAA (0.93\%)\\
8 & CGCC (0.82\%) & CTGC (0.88\%) & GCAA (0.84\%) & CGGC (0.76\%) & CGCC (0.82\%) & CAGC (0.89\%) & CGCA (0.75\%) & CGCC (0.98\%) & CGCC (0.89\%)\\
9 & CAAA (0.82\%) & TTTT (0.85\%) & TGCC (0.84\%) & GAAA (0.75\%) & TTTT (0.82\%) & TGAA (0.87\%) & CTGG (0.74\%) & GTTT (0.97\%) & GGAA (0.89\%)\\
10 & GTTC (0.80\%) & CAGC (0.85\%) & TTCA (0.82\%) & ATGC (0.74\%) & TGGC (0.80\%) & TGCA (0.83\%) & AACA (0.74\%) & TGCC (0.91\%) & AGCA (0.88\%)\\
\cline{1-10}
\rule{0pt}{0.6cm}
  & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Template_top_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Template_top_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Template_top_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Complement_top_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Complement_top_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Complement_top_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_2D_top_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_2D_top_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_2D_top_k4.png} \\
\cline{1-10}
-10 & TAGC (0.10\%) & ACTA (0.11\%) & TTAG (0.10\%) & CGAG (0.13\%) & GTGT (0.13\%) & CTAA (0.10\%) & GGAC (0.12\%) & GACT (0.13\%) & ACTA (0.10\%) & \multirow{10}{*}{\rotatebox[origin=c]{90}{Least common}}\\
-9 & CCCT (0.10\%) & CTAT (0.11\%) & GGAC (0.09\%) & ACTA (0.13\%) & CGAG (0.13\%) & TAGT (0.10\%) & CCCT (0.11\%) & TCTA (0.12\%) & CTAT (0.10\%)\\
-8 & ACTA (0.09\%) & CGAG (0.11\%) & TCTA (0.09\%) & GGAC (0.13\%) & TTAG (0.13\%) & TAGA (0.10\%) & TAGT (0.11\%) & ACCT (0.12\%) & CTAA (0.10\%)\\
-7 & TAGA (0.09\%) & TCTA (0.11\%) & TAGT (0.08\%) & TATA (0.12\%) & TCTA (0.12\%) & TTAG (0.09\%) & TCTA (0.10\%) & CACT (0.12\%) & ACTT (0.10\%)\\
-6 & TCTA (0.07\%) & TAGT (0.10\%) & ACTA (0.08\%) & CTAA (0.12\%) & TAGT (0.11\%) & TAGG (0.07\%) & CTAA (0.09\%) & CGGA (0.11\%) & TATA (0.08\%)\\
-5 & TAGG (0.06\%) & CTAA (0.09\%) & TATA (0.08\%) & CTAT (0.11\%) & CCTC (0.10\%) & ACTA (0.07\%) & TAGG (0.08\%) & TAGG (0.09\%) & CCCT (0.07\%)\\
-4 & CCTA (0.06\%) & TAGA (0.08\%) & TAGA (0.07\%) & GAGG (0.11\%) & ACTA (0.09\%) & CTAT (0.07\%) & TAGA (0.07\%) & TAGA (0.08\%) & TAGA (0.06\%)\\
-3 & TTAG (0.05\%) & TAGG (0.06\%) & CCTA (0.04\%) & CCCT (0.09\%) & CCCT (0.09\%) & CCCT (0.07\%) & CTAT (0.07\%) & CCCT (0.08\%) & TAGG (0.06\%)\\
-2 & TATA (0.04\%) & CCTA (0.03\%) & TAGG (0.04\%) & CCTA (0.06\%) & TAGG (0.08\%) & CCTA (0.03\%) & CCTA (0.04\%) & CCTA (0.06\%) & CCTA (0.04\%)\\
-1 & CTAG (0.02\%) & CTAG (0.00\%) & CTAG (0.01\%) & TAGG (0.04\%) & CCTA (0.06\%) & CTAG (0.01\%) & CTAG (0.03\%) & CTAG (0.01\%) & CTAG (0.00\%)\\
\cline{1-10}
\rule{0pt}{0.6cm}
  & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Template_bottom_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Template_bottom_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Template_bottom_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Complement_bottom_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Complement_bottom_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Complement_bottom_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_2D_bottom_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_2D_bottom_k4.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_2D_bottom_k4.png} \\
\cline{1-10}
\end{tabular}
}
\end{table}
\vspace{-9mm}
{\fontsize{8}{8}\textsf{Kmer space for 4-mers: 256 \hspace{5mm} Random chance for any given 4-mer: 0.39\%}}
\vspace{5mm}
\subsection*{5-mer error motif analysis}
\vspace{-3mm}
\begin{table}[H]
{\footnotesize
\fontsize{6pt}{8pt}\selectfont
\tabcolsep=0.15cm
\begin{tabular}{|c|c c c|c c c|c c c|c}
\cline{1-10}
 & \multicolumn{3}{c|}{Template} & \multicolumn{3}{c|}{Complement} & \multicolumn{3}{c|}{2D} & \\
Rank & Insertion & Deletion & Substitution & Insertion & Deletion & Substitution & Insertion & Deletion & Substitution & \\
\cline{1-10}
1 & CAGCA (0.46\%) & TTGCC (0.42\%) & CAGCA (0.42\%) & CAGCA (0.57\%) & CAGCA (0.42\%) & CAGCA (0.57\%) & CAGCA (0.41\%) & ATTTT (0.71\%) & CAGCA (0.45\%) & \multirow{10}{*}{\rotatebox[origin=c]{90}{Most common}}\\
2 & CATCA (0.40\%) & CAGCA (0.37\%) & GAAAA (0.39\%) & CCAGC (0.40\%) & TCAGC (0.38\%) & ATAAA (0.43\%) & CGGCA (0.37\%) & GAAAA (0.61\%) & CGGCA (0.40\%)\\
3 & ATTTC (0.34\%) & ATTTT (0.34\%) & GCGCA (0.36\%) & CATCA (0.33\%) & ATAAA (0.35\%) & CAGAA (0.38\%) & GCTGG (0.34\%) & GCAAA (0.59\%) & TTGCC (0.37\%)\\
4 & CTGGC (0.34\%) & CAAAA (0.34\%) & CAAAA (0.36\%) & GCTGC (0.31\%) & CTGGC (0.33\%) & CGGCA (0.35\%) & GCAAA (0.32\%) & ATAAA (0.58\%) & GAAAA (0.36\%)\\
5 & TTATC (0.33\%) & GCAAA (0.33\%) & CATCA (0.36\%) & ATAAA (0.31\%) & GCTGC (0.31\%) & CATCA (0.34\%) & CATCA (0.29\%) & TAAAA (0.56\%) & CGCCA (0.36\%)\\
6 & GCAAA (0.31\%) & GCTGC (0.33\%) & AGAAA (0.33\%) & GCGGC (0.31\%) & GCAAA (0.31\%) & CTGCA (0.34\%) & ATTTC (0.29\%) & CAAAA (0.51\%) & CAAAA (0.34\%)\\
7 & GCATC (0.31\%) & GAAAA (0.33\%) & GGAAA (0.32\%) & CGGCA (0.31\%) & ACGCC (0.31\%) & ACGCA (0.32\%) & GCGCA (0.28\%) & CTTTT (0.49\%) & TGGCA (0.32\%)\\
8 & ATTTT (0.31\%) & CATCA (0.32\%) & GCAAA (0.31\%) & ATTTT (0.31\%) & ATTTT (0.31\%) & CGCCA (0.32\%) & TGGCA (0.28\%) & ACAAA (0.43\%) & GCAAA (0.31\%)\\
9 & TTGCC (0.31\%) & CGCCA (0.32\%) & CGGCA (0.31\%) & ACAAA (0.30\%) & CGGCA (0.31\%) & GCTGC (0.32\%) & TCTTC (0.28\%) & TCTTT (0.42\%) & TGAAA (0.31\%)\\
10 & CGCTG (0.30\%) & TGCTG (0.30\%) & TGCTG (0.31\%) & CTGGC (0.29\%) & GAAAA (0.31\%) & TTATC (0.32\%) & TGTTC (0.27\%) & GATTT (0.39\%) & CATCA (0.30\%)\\
\cline{1-10}
\rule{0pt}{0.6cm}
  & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Template_top_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Template_top_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Template_top_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Complement_top_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Complement_top_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Complement_top_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_2D_top_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_2D_top_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_2D_top_k5.png} \\
\cline{1-10}
-10 & ATAGG (0.01\%) & TCTAT (0.01\%) & TAGGT (0.01\%) & ATATA (0.01\%) & CCCCT (0.01\%) & CTAGT (0.01\%) & ACTAG (0.01\%) & GTCCT (0.01\%) & ATAGG (0.01\%) & \multirow{10}{*}{\rotatebox[origin=c]{90}{Least common}}\\
-9 & CGAGT (0.01\%) & CGAGT (0.01\%) & TAGGG (0.01\%) & ACTTG (0.01\%) & GGACT (0.01\%) & CTAGC (0.01\%) & ACCTA (0.00\%) & TAGGA (0.01\%) & CCCTG (0.01\%)\\
-8 & CCGAG (0.01\%) & GGACC (0.01\%) & GGGAC (0.01\%) & CTACT (0.01\%) & CTACT (0.01\%) & TAGAT (0.01\%) & CCCCC (0.00\%) & GCCCT (0.01\%) & CCCTA (0.01\%)\\
-7 & TTTAG (0.01\%) & CTAGT (0.01\%) & CTTGG (0.01\%) & TAGGG (0.01\%) & TAGGG (0.01\%) & CCCTC (0.01\%) & CTAGT (0.00\%) & TCCTA (0.01\%) & GCTAG (0.00\%)\\
-6 & TACTA (0.01\%) & CTTAG (0.01\%) & CTAAC (0.01\%) & TAGGA (0.01\%) & CTAAG (0.01\%) & GGTCC (0.00\%) & CTAGA (0.00\%) & CTAGT (0.01\%) & ACCTA (0.00\%)\\
-5 & CTGAG (0.01\%) & TTGGA (0.01\%) & GTCCA (0.01\%) & GGGAC (0.01\%) & CACTC (0.01\%) & CTAGG (0.00\%) & CTTGT (0.00\%) & CTAGC (0.01\%) & CTAGC (0.00\%)\\
-4 & ACTCG (0.01\%) & GCTAG (0.00\%) & TACTA (0.01\%) & CTAAG (0.01\%) & CTAGT (0.00\%) & CTAGA (0.00\%) & CCTAT (0.00\%) & TCGGA (0.01\%) & CTAGA (0.00\%)\\
-3 & GAGGG (0.01\%) & CTAGC (0.00\%) & TAAGA (0.00\%) & CGAGG (0.01\%) & CTAGC (0.00\%) & TATAG (0.00\%) & CCTAG (0.00\%) & ACTAG (0.01\%) & TAGGG (0.00\%)\\
-2 & GTAGC (0.01\%) & TAGAT (0.00\%) & CTAGC (0.00\%) & TTTAG (0.01\%) & CTAGA (0.00\%) & TAGGT (0.00\%) & TCGGA (0.00\%) & CTAGA (0.00\%) & TAGGA (0.00\%)\\
-1 & ACTAC (0.01\%) & CCCTA (0.00\%) & CCTAT (0.00\%) & GAGGT (0.01\%) & ACCCT (0.00\%) & CCCTA (0.00\%) & ACTAT (0.00\%) & TCTAG (0.00\%) & CTTAG (0.00\%)\\
\cline{1-10}
\rule{0pt}{0.6cm}
  & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Template_bottom_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Template_bottom_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Template_bottom_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_Complement_bottom_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_Complement_bottom_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_Complement_bottom_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_insertion_2D_bottom_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_deletion_2D_bottom_k5.png} & \includegraphics[height=0.5cm]{/home/data/inputs/nanook/main/graphs_last_passonly/motifs/logo_substitution_2D_bottom_k5.png} \\
\cline{1-10}
\end{tabular}
}
\end{table}
\vspace{-9mm}
{\fontsize{8}{8}\textsf{Kmer space for 5-mers: 1024 \hspace{5mm} Random chance for any given 5-mer: 0.10\%}}
\vspace{5mm}
\end{document}
\end{document}
